admin/test/test_images.py

```diff
@@ -375,7 +375,7 @@ def test_help(self):
         ``publish-installer-images`` has a ``--help`` option and includes the
         name of the script in its usage message.
         """
-        returncode, stdout, stderr = self.publish_installer_images(
+        _, stdout, _ = self.publish_installer_images(
```

Comment:
:question: I suggest `_returncode, stdout, _stderr` so that it's clear to future maintainers what the function is returning.